packages/components/molecules/f-skeleton-loader/src/components/skeletons/Sentence.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div class="c-skeleton-loader-sentence c-skeleton-loader-bone">
+        <text-block />
+        <text-block />
+    </div>
+</template>
+
+<script>
+import TextBlock from './Text.vue';
+
+export default {
+    name: 'Sentence',
+    components: { TextBlock }
+};
+</script>
+
+<style lang="scss" scoped>
+  .c-skeleton-loader-sentence .c-skeleton-loader-text:nth-child(2) {
+    max-width: 70%;
+  }
+  .c-skeleton-loader-sentence .c-skeleton-loader-text:not(:last-child) {
+    margin-bottom: 6px;
+  }
```

Comment:
as we're using scss it might make sence to use nesting here:

```
.c-skeleton-loader-sentence {
    .c-skeleton-loader-text:nth-child(2) {
        max-width: 70%;
    }

    .c-skeleton-loader-text:not(:last-child) {
        margin-bottom: 6px;
    }
}
```

Reply:
or even

```
.c-skeleton-loader-sentence {
  .c-skeleton-loader-text {
    &:nth-child(2) {
      max-width: 70%;
    }

    &:not(:last-child) {
      margin-bottom: 6px;
    }
  }
}
```

Reply:
Btw if we remove the wrapping selector (`.c-skeleton-loader-sentence`) will it break anything?